src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -65,21 +66,31 @@ export default class PanoImageRenderer extends Component {
 		this.canvas = this._initCanvas(width, height);
 
 		this._image = null;
-		this._imageLoader = new ImageLoader();
 
-		this._onImageLoad = 	this._onImageLoad.bind(this);
-		this._onImageError = 	this._onImageError.bind(this);
+		this._onContentLoad = 	this._onContentLoad.bind(this);
+		this._onContentError = 	this._onContentError.bind(this);
 
-		this.setImage({image, imageType: sphericalConfig.imageType});
+		this.setImage({image, imageType: sphericalConfig.imageType, isVideo});
 	}
 
-	setImage({image, imageType}) {
```

Comment:
I don't separate a function into video and image. because their logic are almost same.

Reply:
OK